lib/__tests__/Importer-test.js

```diff
@@ -3008,6 +3008,50 @@ bar
         });
       });
     });
+
+    describe('when an import is of the side-effect type', () => {
+      describe('and is by itself', () => {
+        beforeEach(() => {
+          text = `
+import 'bar.html';
+
+bar
+          `.trim();
+        });
+
+        it('is passed through unchanged and sorted to the top', () => {
+          expect(subject()).toEqual(`
+import 'bar.html';
+
+bar
```

Comment:
I think that's what we need to test. The `sorted to the top` is a bit confusing since we only have one import though.